src/scripts/utils.py

```diff
@@ -183,6 +183,18 @@ def _get_next_available_mondo_id(min_id: int, max_id: int, mondo_ids: Set[int])
     return next_id, mondo_ids
 
 
+def remove_angle_brackets(uris: Union[URI, List[URI]]):
```

Comment:
The new `remove_angle_brackets()` func. I've previously written this logic before and been asked by some to leverage some stuff in `curies` or other libraries, so I might refactor to do that. But it is such a simple function to implement.

Reply:
If `semsql make` is provided with all prefixes needed, do you think this will be needed? At that point, the `<>` are ignored by virtue of it's absence in the `curie_map`. On the flip side, some IRIs which may be desired but lost in the process may not be highlighted. @matentzn , thoughts?